src/EnergyPlus/AirflowNetworkBalanceManager.cc

```diff
@@ -8488,6 +8491,7 @@ namespace AirflowNetworkBalanceManager {
                             AirflowNetworkReportData(ZN1).MultiZoneMixSenLossJ +=
                                 (AirflowNetworkLinkReport1(i).FLOW2OFF * CpAir * (MAT(ZN1) - MAT(ZN2))) * ReportingConstant * ReportingFraction;
                         }
+                        // TODO: H2O Heat of Vap?
```

Comment:
Is this still a ToDo or can this comment be deleted?

Reply:
@mjwitte @lgu1234 

This is more of a question than to do. I noticed that in the previous code the latent heat is calculated as
```
AirflowNetworkReportData(ZN1).MultiZoneMixLatGainW += (AirflowNetworkLinkReport1(i).FLOW2OFF * (ZoneAirHumRat(ZN2) - ZoneAirHumRat(ZN1))) * ReportingFraction;
AirflowNetworkReportData(ZN1).MultiZoneMixLatGainJ += (AirflowNetworkLinkReport1(i).FLOW2OFF * (ZoneAirHumRat(ZN2) - ZoneAirHumRat(ZN1))) * ReportingConstant * ReportingFraction;
```

I'm not sure why it does not need to be multiplied by H20 latent heat of evaporation here. I calculated the exfiltration latent heat loss as
```
AirflowNetworkZnRpt(i).ExfilLatentLoss =
                AirflowNetworkZnRpt(i).ExfilMass / ReportingConstant * (ZoneAirHumRat(i) - OutHumRat) * H2OHtOfVap;
```
and I'm not sure if the unit is right here.

Reply:
I agree, this does look wrong.  @lgu1234 new issue posted https://github.com/NREL/EnergyPlus/issues/7162